src/site/_includes/content/lighthouse-accessibility/about-aria.njk

```diff
@@ -1,15 +1,5 @@
-## About ARIA
-
- The [Web Accessibility Initiative's Accessible Rich Internet Applications specification](https://www.w3.org/TR/html-aria)
-(WAI-ARIA, or just ARIA)
-is good for bridging areas with accessibility issues that can't be managed with native HTML.
-It works by allowing you to specify attributes that modify the way an element is translated into the accessibility tree. 
-
- Using ARIA attributes,
-you can give the element the missing information so the screen reader can properly interpret it.
-In order for these attributes to be effective,
-they need to make sense-- you wouldn't want a screen reader
-to misinform a user.
-
- For a brief intro, see the
-[Introduction to ARIA](https://developers.google.com/web/fundamentals/accessibility/semantics-aria/).
\ No newline at end of file
+{% Aside 'key-term' %}
+  ARIA elements let you bridge areas with accessibility issues that can't be managed with native HTML.
+  These elements provide missing information about custom elements so that screen readers can interpret them. 
+  Learn more in this [Introduction to ARIA](https://developers.google.com/web/fundamentals/accessibility/semantics-aria/).
+{% endAside %}
```

Comment:
Mind adding a trailing newline to all of the includes?

Reply:
Done.